routes/activities.routes.js

```diff
@@ -0,0 +1,28 @@
+const router = require("express").Router();
+const ActivityModel = require("../models/Activity.model");
+
+router.get("/activities", async (req, res) => {
+  try {
+    const activities = await ActivityModel.find();
+    res.status(200).json({ activities });
+  } catch (err) {
+    console.error("ERROR fetching activity from DB", err);
+    res.status(500).json({ message: "Internal Server Error" });
+  }
+});
+
+router.post("/new-activity", async (req, res) => {
```

Comment:
We can set the path to `/activities` since `POST` already makes it clear that we are adding something.

Reply:
i did :)